x-pack/plugins/rule_registry/server/routes/get_browser_fields_by_feature_id.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from '@kbn/core/server';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import * as t from 'io-ts';
+
+import { RacRequestHandlerContext } from '../types';
+import { BASE_RAC_ALERTS_API_PATH } from '../../common/constants';
+import { buildRouteValidation } from './utils/route_validation';
+
+export const getBrowserFieldsByFeatureId = (router: IRouter<RacRequestHandlerContext>) => {
+  router.get(
+    {
+      path: `${BASE_RAC_ALERTS_API_PATH}/browser_fields`,
+      validate: {
+        query: buildRouteValidation(
+          t.exact(
+            t.type({
+              featureIds: t.array(t.string),
+            })
+          )
+        ),
+      },
+      options: {
+        tags: ['access:rac'],
+      },
+    },
+    async (context, request, response) => {
+      try {
+        const racContext = await context.rac;
+        const alertsClient = await racContext.getAlertsClient();
+        const { featureIds } = request.query;
+
+        const indices = await alertsClient.getAuthorizedAlertsIndices(featureIds);
+        const o11yIndices = indices?.filter((index) => index.startsWith('.alerts-observability'));
```

Comment:
nit
```suggestion
        const o11yIndices = indices?.filter((index) => index.startsWith('.alerts-observability')) ?? [];
```
I think you will be able to just check the length in the condition